packages/angular/projects/tx-native-angular-sdk/tests/language-picker.component.spec.ts

```diff
@@ -81,34 +71,38 @@ describe('LanguagePickerComponent', () => {
     spyOn(component, 'onChange').and.callThrough();
 
     // act
-    await component.ngOnInit();
+    await component.getLanguages();
```

Comment:
Here I would like to test the `ngOnInit` event, in order to test if the languages are loaded correctly at that stage. Is there any reason for replacing it?